3-probability/06-normal-approx/notes.qmd

```diff
@@ -0,0 +1,841 @@
+---
+title: "Continuous Distributions and Normal Approximations"
+subtitle: "Connections to boxes, continuous distributions, and a fundamental result"
+image: images/front-image.png
+format:
+  html: default
+  pdf: default
+editor_options: 
+  chunk_output_type: console
+---
+
+:::{.lo .content-hidden unless-profile="staff-site"}
+
+#### Concept Acquisition
+
+1. Connection with random variable and draws from a box
+2. IID random variables
+3. Sums and averages of IID random variables
+4. Continuous random variables and their pdf
+5. The unif(a,b) & normal distribution
+6. The Central limit theorem
```

Comment:
My original inclination was to save the CLT for the generalization unit: focusing in the prob section on the normal distribution (where we just say X is approximately normal and go from there) and ask students to, conceptually, be comfortable with `rnorm`, `dnorm` and `pnorm`. We'll see how this goes, though, with CLT here, then we can evaluate afterwards. Now that we're getting these LOs articulated, should be easiest to tell what concepts are where and how to revise things when we wish to.

Reply:
Yes... I haven't said very much, though, nor formalized it. I think it is more a first pass, and then you have to bring it back in the generalization unit. But I am not sure which way is better. Let's see.